From: Marcos Fouces <marcos@debian.org>
Date: Wed, 7 Sep 2022 00:27:23 +0200
Subject: Mark the glyphs as defined in groff.

This prevents the man viewer from issuing
warnings about missing characters.
---
 man/man7/glob.7 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/man7/glob.7 b/man/man7/glob.7
index 05a7f2a..32908b9 100644
--- a/man/man7/glob.7
+++ b/man/man7/glob.7
@@ -195,7 +195,7 @@ where the string between "\fI[=\fP" and "\fI=]\fP" is any collating
 element from its equivalence class, as defined for the
 current locale.
 For example, "\fI[[=a=]]\fP" might be equivalent
-to "\fI[a\('a\(`a\(:a\(^a]\fP", that is,
+to "\fI[a\['a]\[`a]\[:a]\[^a]]\fP", that is,
 to "\fI[a[.a-acute.][.a-grave.][.a-umlaut.][.a-circumflex.]]\fP".
 .SH SEE ALSO
 .BR sh (1),
